src/client/components/page/page.image.vue

```diff
@@ -23,6 +23,10 @@ export default Vue.extend({
 	},
 	created() {
 		this.image = this.page.attachedFiles.find(x => x.id === this.value.fileId);
+
+		if (typeof this.image === "undefined") {
+			this.image = this.value;
+		}
```

Comment:
Is this required?

Reply:
I did it because I did not find a way to convert an Image to a Media or something. When I pass an Image from `page.eyeCatchingImage` it does not have `fileId`, but it's a file and has an `id`, but not `fileId` and in that case it's not even a page attachment just a property of a page.

Any idea how can I resolve this problem without duplicating `x-image`? I would like to fix it better and learn from it as possible.

Reply:
I see.
You can simply use `<img>` tag instead of `<x-image>` in https://github.com/syuilo/misskey/pull/6210/files#diff-b81443d697613ee9cb5072d734e28f73R8

Reply:
Addressed and I don't know why I did not even consider to use raw `img` tag.